Formula/sqlite.rb

```diff
@@ -1,9 +1,9 @@
 class Sqlite < Formula
   desc "Command-line interface for SQLite"
   homepage "https://sqlite.org/index.html"
-  url "https://sqlite.org/2022/sqlite-autoconf-3400000.tar.gz"
-  version "3.40.0"
-  sha256 "0333552076d2700c75352256e91c78bf5cd62491589ba0c69aed0a81868980e7"
+  url "https://sqlite.org/2022/sqlite-src-3400100.zip"
```

Comment:
Any reason why this changed from `sqlite-autoconf` tarball to `sqlite-src` zip?

Reply:
Ah wait, I misunderstood. That was already fixed. Let me change that back.